Tasks/Churo/Task1/add_tasks1.py

```diff
@@ -0,0 +1,5 @@
+initial_amount = 20000
+term = 5
+percent = 15
```

Comment:
these values should be of the decimal data type, since they are money-related, -1 point for the unwise choice